server/text_generation_server/models/flash_causal_lm.py

```diff
@@ -634,6 +634,7 @@ def concatenate(cls, batches: List["FlashCausalLMBatch"]) -> "FlashCausalLMBatch
             next_token_chooser_parameters,
             dtype=batches[0].next_token_chooser.dtype,
             device=batches[0].next_token_chooser.device,
+            tokenizer=batches[0].next_token_chooser.tokenizer
```

Comment:
*here* is where I need access to a tokenizer but one was not conveniently already available in scope.